src/extensions/default/PrefsCodeHints/styles/brackets-prefs-hints.css

```diff
@@ -0,0 +1,29 @@
+.brackets-pref-hints .matched-hint {
+    font-weight: 500;
+}
+.dark .brackets-pref-hints .matched-hint {
+    color: #ccc;
+}
+
+.brackets-pref-hints .hint-obj {
+    display: inline-block;
+    width: 250px;
```

Comment:
Changing this to `min-width 300px` and the below `width` to 300px could be a solution to the horizontal scrollbar problem.